definitions/variable/emissions/emissions.yaml

```
@@ -113,10 +113,9 @@
       - Emissions|{Level-3 Species}|AFOLU|Land|Harvested Wood Products
       - Emissions|{Level-3 Species}|AFOLU|Land|Other
 - Emissions|{Level-3 Species}|AFOLU|Land|Wetlands:
-    description: Emissions from land that is covered or saturated by water for
-      all or part of the year (e.g., peatland) and that does not fall
-      into the forest land, cropland, grassland or settlements
-      categories (partially IPCC 1996 category 4D/5A/5B/5E; IPCC 2006 category 3B4)
+    description: Net emissions from managed wetlands (e.g., peatland), including 
+      emissions from drained wetlands used for agriculture or forestry (IPCC 2006 categories 3B1, 3B2 and 3B3), 
```

These are the precise categories, right?
```suggestion
      emissions from drained wetlands used for agriculture or forestry (IPCC 2006 categories 3B1biii, 3B2biii and 3B3biii), 
```

Just for consistency, what about the other land types converted from wetlands, i.e. 
`3B5biv Wetlands Converted to Settlements`, `3B6biv Wetlands Converted to Other Land`?

---

3B5biv Wetlands Converted to Settlements and 3B6biv Wetlands Converted to Other Land is now included